Dockerfile

```diff
@@ -12,35 +12,34 @@ ARG BUILDTIME
 ARG COMMIT
 ARG TAG
 
-RUN apk add --no-cache \
+RUN apt-get update && apt-get install -y \
   bash \
   make \
   git \
   curl \
   util-linux \
   coreutils \
-  build-base
+  build-essential
 
 # Fetch docker cli to run a registry container for e2e tests
 RUN curl -Ls https://download.docker.com/linux/static/$DOCKERCLI_CHANNEL/x86_64/docker-$DOCKERCLI_VERSION.tgz | tar -xz
 
 # Fetch docker-app to build a CNAB from an application template
-RUN curl -Ls https://github.com/docker/app/releases/download/$DOCKER_APP_VERSION/docker-app-linux.tar.gz | tar -xz
-RUN git clone https://github.com/docker/app
+RUN curl -Ls https://github.com/docker/app/releases/download/$DOCKER_APP_VERSION/docker-app-linux.tar.gz | tar -xz && cp /go/docker-app-linux /usr/bin/docker-app
```

Comment:
We should bump the version we use to the lastest release of Docker App. I see above that it's the cnab-preview one